pkg/agent/cniserver/pod_configuration.go

```diff
@@ -259,14 +263,18 @@ func (pc *podConfigurator) configureInterfaces(
 	return nil
 }
 
-func (pc *podConfigurator) createOVSPort(ovsPortName string, ovsAttachInfo map[string]interface{}) (string, error) {
+func (pc *podConfigurator) createOVSPort(ovsPortName string, ovsAttachInfo map[string]interface{}, vlanID uint16) (string, error) {
 	var portUUID string
 	var err error
 	switch pc.ifConfigurator.getOVSInterfaceType(ovsPortName) {
 	case internalOVSInterfaceType:
 		portUUID, err = pc.ovsBridgeClient.CreateInternalPort(ovsPortName, 0, ovsAttachInfo)
 	default:
-		portUUID, err = pc.ovsBridgeClient.CreatePort(ovsPortName, ovsPortName, ovsAttachInfo)
+		if vlanID == 0 {
```

Comment:
Yes. I prefer to add comment at https://github.com/antrea-io/antrea/pull/3247#discussion_r828981232 since this is the first time we use uint16 for VLAN